logstash-core/lib/logstash/pipeline.rb

```diff
@@ -37,6 +37,8 @@ module LogStash; class Pipeline
     :config_str,
     :original_settings
 
+  DEFAULT_OUTPUT_WORKERS = 1
+
   DEFAULT_SETTINGS = {
```

Comment:
I'd rather not have defaults in there in general. I think we should probably pull `default_pipeline_workers` out of there IMHO.